docs/reference/README.md

```diff
@@ -1,18 +1,104 @@
 # Knative API Reference documentation
 
+The reference documentation for the latest release of the Knative is available
+at [www.knative.dev](https://www.knative.dev/docs/reference/).
+
+Use the following list to view the "pre-release" source files:
+
 - [Serving API](./serving.md)
 - [Build API](./build.md)
 - [Eventing API](./eventing/eventing.md)
 - [Event Sources API](./eventing/eventing-sources.md)
 
-Note: The Knative API reference documentation is manually generated using the
-[gen-api-reference-docs.sh](../../hack/) tool. After cloning the Knative/docs
-repo, you must open the `gen-api-reference-docs.sh` script locally and specify
-the commit/tag for each of the Knative repos before running that script.
-To build a specific version of the API, you must specify the corresponding
-commit or tag numbers across each of the Knative repos.
-For example, to build a set of Knative API docs for v0.3, you can the tag
-numbers in each repo. So for
-[Serving v0.3.0](https://github.com/knative/serving/tree/v0.3.0) you specify
-`KNATIVE_SERVING_COMMIT="v0.3.0"`, and so on for each of the Build, Eventing,
-and Eventing Sources repos.
+
+## Updating API Reference docs (for Knative maintainers)
+
+The Knative API reference documentation is manually generated using the
+[`gen-api-reference-docs.sh`](../../hack/) tool. If you need to generate a new
+version of the API docs for a recent update or for a new release, you can use
+the steps below.
+
+To learn more about the tool, see the
+[gen-crd-api-reference-docs](https://github.com/ahmetb/gen-crd-api-reference-docs)
+reference page.
+
+### Before you begin
+
+You must meet the following requirements to run the `gen-api-reference-docs.sh`
+tool:
+
+* You need the following software installed:
+  * [`git`](https://git-scm.com/download/)
+  * [`go` version 1.11+](https://golang.org/dl/)
+* You must locally clone the [knative/docs](https://github.com/knative/docs)
+  repository. For example: `git clone git@github.com:knative/docs.git`
+
+### Generating the API
+
+To generate a version of the API:
+
+1. Ensure that your `GOPATH` excludes a value. The `gen-api-reference-docs.sh`
```

Comment:
In this case, "excludes" feels a bit odd. What about "doesn't have?"

Reply:
how about "is empty"